cmd/locator/main.go

```diff
@@ -3,54 +3,47 @@ package main
 import (
 	"context"
 	"fmt"
-	"os"
 
-	log "github.com/noxiouz/zapctx/ctxlog"
```

Comment:
why?

Reply:
There is no point in adding extra aliases. This exact package is used once only.